index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Mobile Portfolio</title>
+  <div class="header-items">
+    
+    <img src="img/My Logo.svg" alt="humburger" class="Logo" > 
+    <img src="img/Union.svg" alt="humburger"  class="Union" >
```

Comment:
- Your logo and hamburger should be in a header tag. This is because a header tag represents the introductory content of your page.
- For semantic purposes and to also follow the HTML convention, kindly consider wrapping your hamburger in a nav tag considering it is also used for navigation.
-  Kindly apply the appropriate margin in your header which I have highlighted in the image below because it is currently not looking like that of the Figma.

Your Design
![image](https://github.com/Edgahkipkemoi/mobile/assets/89091822/ecb06cd2-ac69-445c-bff1-4779f8a4f8fa)

Figma Design
![image](https://github.com/Edgahkipkemoi/mobile/assets/89091822/afcc4b0f-b619-481d-a674-5d0094ec034a)